src/platform/web/ui/css/themes/element/theme.css

```diff
@@ -43,14 +41,37 @@ limitations under the License.
     color: white;
 }
 
-.hydrogen .avatar.usercolor1 { background-color: var(--usercolor1); }
-.hydrogen .avatar.usercolor2 { background-color: var(--usercolor2); }
-.hydrogen .avatar.usercolor3 { background-color: var(--usercolor3); }
-.hydrogen .avatar.usercolor4 { background-color: var(--usercolor4); }
-.hydrogen .avatar.usercolor5 { background-color: var(--usercolor5); }
-.hydrogen .avatar.usercolor6 { background-color: var(--usercolor6); }
-.hydrogen .avatar.usercolor7 { background-color: var(--usercolor7); }
-.hydrogen .avatar.usercolor8 { background-color: var(--usercolor8); }
+.hydrogen .avatar.usercolor1 {
```

Comment:
I imagine your tooling got ahead of you here, but please don't make unrelated stylistic changes like this. For one, it makes it harder to see where you made actual changes in this file.